lisa/util/shell.py

```diff
@@ -101,6 +104,14 @@ def wait_tcp_port_ready(
     return is_ready, result
 
 
+def _is_valid_ipv6(address: str) -> bool:
+    try:
+        ipaddress.IPv6Address(address)
```

Comment:
Check IPv4, so it won't raise exception on most scenarios. Raise exceptions is much higher cost the if/else.